PRD-mfa-enforcement.md

```
@@ -0,0 +1,1108 @@
+    amr: claims.amr ?? [],
+  };
+}
+```
+
+`AuthSession` in [server/session.ts](apps/webapp/server/session.ts) gains two new fields: `aal: 'aal1' | 'aal2'` and `amr: { method: string; timestamp: number }[]`.
+
+### 4.3 Middleware: `enforceMfa()`
+
+New middleware between `protect()` and the route handlers in [server/index.ts](apps/webapp/server/index.ts):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)